driver-core/src/main/com/mongodb/operation/AggregateToCollectionOperation.java

```diff
@@ -100,23 +103,54 @@ public AggregateToCollectionOperation(final MongoNamespace namespace, final List
      */
     public AggregateToCollectionOperation(final MongoNamespace namespace, final List<BsonDocument> pipeline,
                                           final WriteConcern writeConcern) {
-        this(namespace, pipeline, writeConcern, AggregationLevel.COLLECTION);
+        this(namespace, pipeline, null, writeConcern, AggregationLevel.COLLECTION);
     }
 
     /**
      * Construct a new instance.
      *
      * @param namespace the database and collection namespace for the operation.
      * @param pipeline the aggregation pipeline.
+     * @param readConcern the read concern to apply
+     *
+     * @since 3.11
+     */
+    public AggregateToCollectionOperation(final MongoNamespace namespace, final List<BsonDocument> pipeline,
+                                          final ReadConcern readConcern) {
+        this(namespace, pipeline, readConcern, null, AggregationLevel.COLLECTION);
+    }
+
+    /**
+     * Construct a new instance.
+     *
+     * @param namespace the database and collection namespace for the operation.
+     * @param pipeline the aggregation pipeline.
+     * @param writeConcern the write concern to apply
+     * @param readConcern the read concern to apply
+     * @since 3.11
+     */
+    public AggregateToCollectionOperation(final MongoNamespace namespace, final List<BsonDocument> pipeline,
+                                          final ReadConcern readConcern, final WriteConcern writeConcern) {
+        this(namespace, pipeline, readConcern, writeConcern, AggregationLevel.COLLECTION);
+    }
+
+    /**
+     * Construct a new instance.
+     *
+     * @param namespace the database and collection namespace for the operation.
+     * @param pipeline the aggregation pipeline.
+     * @param readConcern the read concern to apply
      * @param writeConcern the write concern to apply
      * @param aggregationLevel the aggregation level
      * @since 3.10
      */
     public AggregateToCollectionOperation(final MongoNamespace namespace, final List<BsonDocument> pipeline,
```

Comment:
So we need to keep the 3.10 constructor as is: 
```
public AggregateToCollectionOperation(final MongoNamespace namespace, final List<BsonDocument> pipeline,
                                      final WriteConcern writeConcern, final AggregationLevel aggregationLevel)```

And add the new one that also contains ReadConcern.

Reply:
Done